test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -4336,3 +4336,22 @@ Describe 'Invoke-WebRequest and Invoke-RestMethod support Cancellation through C
         RunWithCancellation -Command 'Invoke-RestMethod' -Uri $uri
     }
 }
+
+Describe "Web cmdlets Unix Sockets tests" -Tags "CI", "RequireAdminOnWindows" {
+    BeforeAll {
+        $WebListener = Start-UnixSocket
+    }
+
+    It "Execute Invoke-WebRequest with -UnixSocket" {
+        $uri = Get-UnixSocketUri
+        $result = Invoke-WebRequest $uri -UnixSocket "/tmp/foo.sock"
```

Comment:
We shouldn't use `foo` in names. Please user better name.
Also it is better to get the test socket name from `Get-UnixSocketUri`  (or new Get-UnixSocketName).

Reply:
Done